src/redux/modules/userPatch/index.js

```diff
@@ -0,0 +1,40 @@
+import { useDispatch, useSelector } from 'react-redux'
+import { useCallback } from 'react'
+import { createSlice } from '@reduxjs/toolkit'
+
+const initialState = {
+  data: null,
+  error: null,
+}
+
+const patchSlice = createSlice({
```

Comment:
In case of having data, error and loading you can still use redux-fetch-state module